stubs/fpdf2/METADATA.toml

```diff
@@ -1,5 +1,5 @@
-version = "2.6.0"
-requires = ["types-Pillow"]
+version = "2.6.1"
+requires = ["cryptography", "types-Pillow"]
```

Comment:
Doesn't look like the `cryptography` dependency is being used anywhere

```suggestion
requires = ["types-Pillow"]
```

Reply:
Moved to correct file.